utils/build.ps1

```diff
@@ -3638,7 +3638,7 @@ function Build-ArgumentParser([Hashtable] $Platform) {
     -Bin (Get-ProjectBinaryCache $Platform ArgumentParser) `
     -InstallTo "$($Platform.ToolchainInstallRoot)\usr" `
     -Platform $Platform `
-    -UseBuiltCompilers Swift `
+    -UseBuiltCompilers Swift,C `
```

Comment:
question: why is this need to bump the `swift-argument-parser` version?

Reply:
Who did you talk with? I want to make sure that they are on the same page as me. The DocC issues should be resolved at this point. I think that it would be better to use the MSVC compilers for the time being to maintain a uniform behaviour throughout the build.